frontend/openchat-client/src/openchat.ts

```diff
@@ -3074,30 +3075,17 @@ export class OpenChat extends EventTarget {
             : undefined;
 
         if (eventsResponse === undefined || eventsResponse === "events_failed") {
-            return false;
+            return;
         }
 
         await this.#handleEventsResponse(serverChat, eventsResponse);
-        // We may have loaded messages which are more recent than what the chat summary thinks is the latest message,
-        // if so, we update the chat summary to show the correct latest message.
-        const latestMessage = findLast(eventsResponse.events, (e) => e.event.kind === "message");
-        const newLatestMessage =
-            latestMessage !== undefined && latestMessage.index > serverChat.latestEventIndex;
-
-        if (newLatestMessage) {
-            updateSummaryWithConfirmedMessage(
```

Comment:
This now takes place within `handleEventsResponse`, which makes more sense because it should happen whenever we load an event window or load new events